### ISSUE TITLE
Extend `-print-transformations`

### ISSUE THREAD

**Issue by Kakadu** [ocaml-ppx/ppxlib#18]
I'm currently developing a deriving rewriter which is extending `type-conv` and is being registered using `Type_conv.add "name"`. When I run `-print-transformations` the standalone rewriter gives me only `type_conv`. It would be great if it will print all the generators registered for type_conv.

P.S. I actually have a bug at the moment that standalone rewriter is not linked with my code and complains like `Error: ppxlib_type_conv: 'name' is not a supported type type-conv generator`. It would be great to know for sure (using switch above) that the library that registers generator is not linked.

**Comment by diml** [comment]
Yes, that makes sense. Currently `type_conv`  is seen as a single rewriter by the rest of the machinery, so it's not straightforward to support this feature. It might be easier after #14.

For the bug, could you open a separate issue with a bit more information about how to reproduce it?
